pyfar/utils.py

```diff
@@ -212,3 +212,45 @@ def concatenate_channels(signals, caxis=0, broadcasting=False):
         return pf.TimeData(data, signals[0].times)
     else:
         return pf.FrequencyData(data, signals[0].frequencies)
+
+
+def concatenate_bins(signals):
+    """
+    Merge multiple FrequencyData objects along the frequency axe.
+
+    Parameters
+    ----------
+    signals : tuple of FrequencyData
+        The signals to concatenate. All signals must have the same cshape.
+        the frequency bins get sorted and doubles are removed, the first
+        entry is used.
+
+    Returns
+    -------
+    merged : FrequencyData
+        The merged signal object.
+    """
+    # check input
+    if not isinstance(signals, (tuple, list)):
+        raise TypeError(
+            "Input must be a tuple or list of pf.FrequencyData objects.")
+
+    for signal in signals:
+        if not isinstance(signal, (pf.FrequencyData)):
```

Comment:
This is `True` for `pyfar.Signal` due to the inheritance. I think `type(signal) != pf.FrequencyData` is what you want.

Reply:
This should also be tested then.